index.html

```diff
@@ -8,83 +8,321 @@
+            </div>
+            <div class="project-tag">
+              <ul class="project-skills flex">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <button>See Project</button>
+          </div>
+        </div>
+
+        <div class="project-card flex">
+          <img
+            src="./images/Snapshoot Portfolio (2).svg"
+            alt="project card image"
+          />
+          <div class="prject-card-info flex column">
+            <h2 class="project-title">Tonic</h2>
+            <div class="project-meta flex">
+              <p class="bold">Canopy</p>
+              <div class="round"></div>
+              <p>Backend</p>
+              <div class="round"></div>
+              <p>2015</p>
+            </div>
+            <div class="project-note">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </div>
+            <div class="project-tag">
+              <ul class="project-skills flex">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <button>See Project</button>
+          </div>
+        </div>
+
+        <div class="project-card flex">
+          <img
+            src="./images/Snapshoot Portfolio (3).svg"
+            alt="project card image"
+          />
+          <div class="prject-card-info flex column">
+            <h2 class="project-title">Tonic</h2>
+            <div class="project-meta flex">
+              <p class="bold">Canopy</p>
+              <div class="round"></div>
+              <p>Backend</p>
+              <div class="round"></div>
+              <p>2015</p>
+            </div>
+            <div class="project-note">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </div>
+            <div class="project-tag">
+              <ul class="project-skills flex">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <button>See Project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section id="about">
+      <div class="about-container flex column">
+        <div class="abt-me flex column">
+          <h2 class="header-text">About Myself</h2>
+          <p class="para-text">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <div class="self-container">
+            <p>Let's Connect</p>
```

Comment:
- [ Optional ] Kindly i Recommend you use a header tag `hx` for  titles (e.g., "Let's connect"), This way it will be semantic correct.